spec/migrations/20180626125655_rename_user_maintenance_zone_record_spec.rb

```diff
@@ -0,0 +1,74 @@
+require_migration
+
+describe RenameUserMaintenanceZoneRecord do
+  let(:zone_stub) { migration_stub(:Zone) }
+
+  let(:remote_region_start) do
+    anonymous_class_with_id_regions.rails_sequence_start +
+      anonymous_class_with_id_regions.rails_sequence_factor
+  end
+
+  migration_context :up do
+    before(:each) do
+      zone_stub.delete_all
```

Comment:
hm, this should not happen, something must be off, because it should all just a transaction that gets rolled back

Delete like this will not work if the specs will run in parallel. So we need to find a way to to properly fix this

Reply:
@Ladas probably some bad `before :all` in the past in my local db so it work without it normally.